src/server/pachyderm_test.go

```diff
@@ -8841,6 +8843,26 @@ func TestDeleteSpecRepo(t *testing.T) {
 	require.YesError(t, err)
 }
 
+func TestDeleteRepo(t *testing.T) {
```

Comment:
Can we move this test to one of the files in `/src/server/pfs/server/testing`?

Reply:
moved to "server_test.go"